docs/DeveloperGuide.md

```diff
@@ -233,6 +233,24 @@ implement `Predicate<Student`, and predicate class for filtering `TuitionClass`
 Each custom predicate class contains a `List` of search strings that would be used to match against the tested items 
 in the search.
 
+### Class Panel feature
```

Comment:
Do you want to name it `Classes` feature instead? To match the name of our tabs

Reply:
Ok I'll change the name to Classes Ui feature, Classes feature sounds a bit too vague